main.cpp

```diff
@@ -137,6 +138,24 @@ void Phone::displayInitState()
               << "Thickness = " << thickness << std::endl;
 }
 
+void Phone::recalculateMemory(int memory)
+{
+    int iterator = 0;
+    std::cout << "Doubling memory until it reaches max amount\n";
+    
+    while (memory < memoryAmount)
+        {
+            ++iterator;
+            memory *= 2;
+            std::cout << "Memory is: " << memory << " of maximum " << memoryAmount << std::endl;
+        }
+    
+    for (int i = 0; i < iterator; ++i)
+        {
+            std::cout << "Calculation applied " << i + 1 << "th time of " << iterator << std::endl;
+        }
```

Comment:
FIXME: indentation
curly braces should line up with the keywords on the line above
- [ ] revise